src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/Touch/GestureManager.shared.cs

```diff
@@ -109,7 +109,7 @@ internal async Task ChangeStateAsync(TouchBehavior sender, bool animated, Cancel
 			}
 			catch (TaskCanceledException ex)
 			{
-				Trace.WriteLine(ex);
+				Trace.TraceError(ex.StackTrace);
```

Comment:
I think we should rely on the full information

```suggestion
				Trace.TraceError(ex);
```